src/databricks/labs/remorph/reconcile/query_builder/base.py

```diff
@@ -110,6 +113,13 @@ def _get_column_alias(layer: str, col: str, col_mapping: dict[str, ColumnMapping
 
         return col_origin, col_alias
 
+    @staticmethod
+    def _generate_join_condition(columns: set[str]):
```

Comment:
Can't this be generated as part of expression ?

Reply:
Stale code. Its already part of expression_generator. Removed it.